### DISCUSSION TITLE
Purina My Felix the Desktop Cat

### DISCUSSION THREAD

**Discussion by nima-amin** [ruffle-rs/ruffle#4171] [General] [answered]
Since My Felix has stopped running in January, it seems to have to do something wit Flash even though it is an .exe file.

Will there be any chance to make it run in/with Ruffle?

The program exists in German and English. The German file I have (in case anybody needs it to try his hands on it), the English file you can find here: https://archive.org/details/myfelix

You can see him in action here: https://www.youtube.com/watch?v=czK2hKkpcFs

I would love to have my little guy back on board.

**Comment by DaVince** [comment]
This looks like a very specific case where it expects a specific plugin for a specific browser. I doubt that would be implemented through Ruffle and if it did it's no guarantee this application is going to work.

If I were you I'd instead try to track down Flash Player 8 for Internet Explorer and install that.

**Comment by flashliberty5467** [comment]
i just filed a feature request referencing this page on this issue

**Comment by nima-amin** [comment]
@DaVince

Well, it's July now which means that with this month's patch day Microsoft is deleting the system implemented Flash Player (the ax one for Internet Explorer) in Windows 10 and 8 and installs something that prevents re-installation, so it seems to be a bit trickier than just install one of the last FP versions (or even very early ones like FP 8 - seriously???) that still worked (I have the Mozilla FP version running in Pale Moon, but wonder what will happen to that after the purge through Microsoft since all the Flash Player Plugins are stored in the same System Folder - which I fear Microsoft will simply erase with all content, even though they said long ago they will leave user-installed versions - like the Mozilla Plugin - untouched). The Standalone Flash Player version won't help here, since as you pointed out it needs this browser thingy even though it doesn not use the browser at all.
In a coming step Microsoft is also planning to erase Internet Explorer from the systems from 15th July 2022 onwards.

@flashliberty5467

Thank you!

**Comment by flashliberty5467** [comment, marked as answer]
i got a link to a guide for your screensaver situation created by n0samu 
https://nosamu.medium.com/how-to-reactivate-flash-based-games-and-screensavers-on-windows-10-or-8-1-f9b0218a332c

**Comment by nima-amin** [comment]
@flashliberty5467

Wow, this sounds great, thanks, however:

1) I don't have the FP version from China but version 32.0.0.445, so which version of the Patcher do I have to use?
2) Is this tool and its use safe?
3) Will that still work after Microsoft did what it announces here:
https://blogs.windows.com/msedgedev/2020/09/04/update-adobe-flash-end-support/

**Reply by nima-amin** [reply]
@flashliberty5467

THANK YOU SO MUCH for this. My little Felix is back on board and I can also use SWFPlayer and the Flash Player Plugin of Irfan View again.

Maybe it was a good idea to wait until Microsoft had erased the original system-inbuilt Active-X Flash Player since I read that several people had problems to overwrite the ocx.vch file.  Tonight a cumulative update (was that already Patch Day or something extra, I don't know) was applied and after restarting the PC I checked the Macromed folders and saw the Active X Flash Payer version was gone, so I dared try to install this Clean FP and it worked. No errors and Felix started.
